### PR TITLE
Fix an issue where unified integrations / scripts with a period in th…

### PR DESCRIPTION
**Pull request by MichaelYochpaz**
## Related Issues
This PR fixes the issue mentioned [here](https://github.com/demisto/demisto-sdk/pull/3295/files#r1290410676), which seems to be mainly noticeable when downloading content.

## Description
This PR fixes an issue when splitting integrations with a period in their name (ex: "Test v1.0") would result in the python file being named "Test v1.py" due to `.0` being detected as a suffix by `pathlib.Path`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 * Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with new lines followed by square/curly brackets.
 * Fixed an issue where **graph** commands would not clean their temporary files properly, causing successive commands to fail.
 * Fixed an issue where an error log message changed the terminal color.
+* Fixed an issue where in some cases, splitting unified integrations / scripts with a period in their name would result in an incorrect filename being used for the code file.
```
Comment (GuyAfik) [inline CHANGELOG.md:12]: which cases? can it be more specific and still be a short log?

move it to `Unreleased`

File: demisto_sdk/tests/test_files/integration-Zoom-v1.0.yml [hunk 2]
```diff
@@ -0,0 +1,409 @@
+commonfields:
+  id: Zoom-v1.0
+  version: -1
```
Comment (GuyAfik) [inline demisto_sdk/tests/test_files/integration-Zoom-v1.0.yml:3]: can we avoid adding new test files?
can you use `TestSuite` here instead maybe?

File: CHANGELOG.md (hunk 1 above)
Comment (MichaelYochpaz) [reply CHANGELOG.md]: Updated it, let me know what if this is better.

File: demisto_sdk/tests/test_files/integration-Zoom-v1.0.yml (hunk 2 above)
Comment (MichaelYochpaz) [reply demisto_sdk/tests/test_files/integration-Zoom-v1.0.yml]: Fixed on c9d835df9f7357f35ef6dac69b6e88e9054fb9ae

**Pull request merged**